syntax = "proto3";

package sound;

message SpectrogramConfig {
  string window_type = 1;
  int64 nperseg = 2;
  int64 noverlap = 3;
  int64 nfft = 4;
}

